ui/src/app/streams/stream-deploy/free-text/free-text.component.html

```diff
@@ -0,0 +1,30 @@
+<form [formGroup]="formGroup" (submit)="deployStream()">
+  <p>Enter the list of properties into the text area field below. Alternatively, you can also select a file in your local file system, which is used to populate the text area field.</p>
+  <div class="form-textarea">
+    <div class="numbers">
+      <ng-template ngFor let-item [ngForOf]="lines">
+        <div class="number">
+          <span [class.invalid]="!item.valid">{{ item.label }}</span>
+        </div>
+      </ng-template>
+    </div>
+    <textarea autosize autofocus formControlName="input"></textarea>
```

Comment:
This also raises the question: Should we prefix all our directives? In the predecessor it was easy as it was called "Spring XD", so we could prefix everything with `xd`. Maybe `scdf` or just `df` (Data Flow).